NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java

```diff
@@ -295,6 +295,16 @@ private static void useFasterCannonTracker() {
         c.addComment("settings.faster-cannon-tracker", "Enables a faster cannon entity tracker");
     }
 
+    public static boolean disableTracking;
+    public static int trackingThreads;
+
+    private static void disableTracking() {
```

Comment:
Why both configs in this function? Do this instead:
```suggestion
    private static void tracking() {
```